src/postgres/getCatalog.js

```diff
@@ -75,7 +75,22 @@ const addTables = (client, catalog) =>
     select
       n.nspname as "schemaName",
       c.relname as "name",
-      d.description as "description"
+      d.description as "description",
+      case when
+           (pg_catalog.pg_relation_is_updatable(c.oid, true)::bit(8) & B'00010000') = B'00010000'
```

Comment:
Another option is we could try to do a mock insert/update/delete and observe the error codes. So something like:

```sql
begin;
insert into non_updatable values (1);
rollback;
```

And if a `55000` is thrown ([error codes](https://www.postgresql.org/docs/9.1/static/errcodes-appendix.html)) we know we can’t insert. This is just an idea, until I see the `pg_relation_is_updatable` docs I can’t be certain which method is better so I defer this decision to you.